.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: RELEASE
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  test:
+    name: Release ${{ matrix.node }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ["14.x"]
+        os: [ubuntu-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Installing Dependencies
+        run: npm install
+
+      - name: Running Tests
+        run: npm test
+
+      - name: Publish
+        run: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc && npm publish --access public
```

Comment:
Instead of that. Let's configure `registry-url` at `actions/setup-node@v1` (as e.g. here: https://github.com/serverless/template-package/blob/50f01db83c28c7b7c8e8437564efe9fe73bd3434/.github/workflows/publish.yml#L35)

Reply:
Done